odl/phantom/transmission.py

```diff
@@ -65,12 +71,14 @@ def _modified_shepp_logan_ellipses(ellipses):
     """
     intensities = [1.0, -0.8, -0.2, -0.2, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
 
+    assert len(ellipses) == len(intensities)
```

Comment:
Not to this one, it's basically a utility so we wont have to write two functions that create the modified ellipses. If a user wants to use this the correct usage would be something like

```python
ellipses = shepp_logan_ellipses(3, modified=True)
# modify ellipses however you want
```